CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## vUNRELEASED
+## v1.1.0
+
+This release is the first incremental release since 1.0. It includes bugfixes and some UX improvements.
 
 #### New features and UX changes
 
 - Add support for non-numerical UID/GID as specified in the appc spec ([#2159](https://github.com/coreos/rkt/pull/2159)).
 - When an application terminates with a non-zero exit status, `rkt run` now returns that exit status ([#2198](https://github.com/coreos/rkt/pull/2198)). This requires [systemd >= v227](https://lists.freedesktop.org/archives/systemd-devel/2015-October/034509.html) in stage1. systemd-v229 can now be used in the [src and host flavors](https://github.com/coreos/rkt/blob/master/Documentation/build-configure.md#--with-stage1-flavors) but this is not yet available in the default coreos flavor.
```

Comment:
Since it will not work in the released binary, I would like to word this a bit differently to make clear it is a partial fix for now...

> When an application terminates with a non-zero exit status, `rkt run` should return that exit status. This is now partially fixed in the [src and host flavors](https://github.com/coreos/rkt/blob/master/Documentation/build-configure.md#--with-stage1-flavors) with [systemd >= v227](https://lists.freedesktop.org/archives/systemd-devel/2015-October/034509.html) ([#2198](https://github.com/coreos/rkt/pull/2198)) but not yet fixed in the default coreos flavor.